app/templates/components/forms/admin/settings/billing.hbs

```diff
@@ -0,0 +1,77 @@
+<form class="ui form" {{action 'submit' on='submit'}}> 
```

Comment:
Shouldn't there be an `ui form {{if isLoading 'loading'}}` for a loading UI?

Reply:
It's there in `app/templates/admin/settings/billing.hbs`

Reply:
@mrsaicharan1 Oh! alright, missed it.